examples/lbfgs/driver.py

```diff
@@ -1,3 +1,7 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
Similarly here, this doesn't seem necessary for a plain `print()`... you'd need it if you're passing an output stream or something though.